models/corporation-manager.go

```diff
@@ -11,28 +11,18 @@ import (
 type CorporationManagerAuthentication struct {
 	User     string `json:"user"`
 	Password string `json:"password"`
-	Platform string `json:"platform"`
-	Org      string `json:"org"`
-	Repo     string `json:"repo"`
+	dbmodels.OrgRepo
 }
 
-func (this CorporationManagerAuthentication) IsValidate() bool {
-	if this.Platform == "" || this.Org == "" {
-		return false
+func (this CorporationManagerAuthentication) Validate() IModelError {
+	if this.Platform == "" || this.OrgID == "" {
+		return newModelError(ErrEmptyPayload, fmt.Errorf("platfrom or orgID is empty"))
 	}
-	return true
-}
-
-func (this CorporationManagerAuthentication) OrgIdentity() string {
-	po := fmt.Sprintf("%s/%s", this.Platform, this.Org)
-	if this.Repo == "" {
-		return po
-	}
-	return fmt.Sprintf("%s/%s", po, this.Repo)
+	return nil
```

Comment:
if this.OrgRepoID() == "" {
return newModelError(ErrEmptyPayload, fmt.Errorf("platfrom or orgID is empty"))
}

Reply:
why not validate password and user

Reply:
Cannot use this.OrgRepoId()method to judge, because this method will never return ""

Reply:
i think pwd and user no need to judge here because Already dealt with [here](https://github.com/opensourceways/app-cla-server/blob/93a1aee05b8af1630077c4bf87bde164f2787e51/controllers/auth_on_corp_manager.go#L30)